lib/DDG/Spice/JrDevJobs.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::JrDevJobs;
+# ABSTRACT: Search for Junior Developer Jobs from JrDevJobs.com
+
+use DDG::Spice;
+
+triggers start => qw(jrdevjobs);
```

Comment:
using `startend` here would be fine. That way you can handle queries like "javascript developer jrdevjobs"